src/mongoc/mongoc-server-description-private.h

```diff
@@ -104,6 +94,10 @@ mongoc_server_description_set_set_version (mongoc_server_description_t *descript
 void
 mongoc_server_description_set_election_id (mongoc_server_description_t *description,
                                            const bson_oid_t            *election_id);
+
+const char *
+mongoc_server_description_type_string (const mongoc_server_description_t *description);
```

Comment:
Existing private functions in this file didn't use an underscore prefix, so I didn't use it. I can change this in the second commit if you decide that [CDRIVER-1198](https://jira.mongodb.org/browse/CDRIVER-1198) is worthwhile.